torch_geometric/nn/models/graph_cross_network.py

```diff
@@ -0,0 +1,588 @@
+import torch
+import torch.nn
+from torch import Tensor
+import torch.nn.functional as F
+from torch_geometric.data import Batch
+from torch_geometric.utils import subgraph
+from torch_geometric.nn.conv import GCNConv
+from torch_geometric.nn.glob import global_sort_pool
+
+from typing import List, Optional, Union
+
+
+class GCNConvWithDropout(torch.nn.Module):
```

Comment:
Let's not create two much modules here :)

For example, `GCNConvWithDropout`, `DenseLayer` can be safely dropped, IMO.